packages/reporter-dom/src/index.ts

```diff
@@ -52,6 +53,7 @@ function setValidationMessage(
     spanElement.dataset.felteReporterDomSingleMessage = '';
     const textNode = document.createTextNode(validationMessage);
     spanElement.appendChild(textNode);
+    spanElement.classList.add(singleClass);
```

Comment:
I'm not sure if passing an empty string to this is why the tests are failing?

From what I see on [mdn](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) the `add` function accepts a class per argument. Maybe also accepting an array would be nice. So the type definition can be `string | string[]` and to check for emptiness before calling `add`.